### PR TITLE
Pagination, Sorting, and Swagger task

### PR DESCRIPTION
**Pull request by Stelmaschyk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,160 +1,159 @@
...
-				<artifactId>liquibase-maven-plugin</artifactId>
-				<version>${liquibase.version}</version>
-				<configuration>
-					<propertyFile>src/main/resources/liquibase.properties</propertyFile>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```
Comment (Elena-Bruyako) [inline pom.xml:4]: why there are lots of changes in pom file?

File: src/main/java/mate/academy/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -19,42 +25,57 @@
...
+    @Operation(summary = "Delete book by id")
     @DeleteMapping("/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void deleteBookById(@PathVariable Long id) {
         bookService.deleteById(id);
     }
 
+    @Operation(summary = "Search books by author and title parameters only")
     @GetMapping("/search")
     public List<BookDto> searchBook(BookSearchParametersDto searchParameters) {
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/bookstore/controller/BookController.java]: add Pageable

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

File: pom.xml (hunk 1 above)
Comment (Stelmaschyk) [reply pom.xml:4]: Everything is banal, I can’t resolved mergeConflict, having already copied the preparations of the POM.

**Review by Elena-Bruyako — APPROVED** [review]

**Pull request merged**